tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -635,4 +651,30 @@ function() {
 		$tag = wp_get_attachment_image( $attachment_id, 'medium', false, array( 'class' => "wp-image-{$attachment_id}" ) );
 		$this->assertNotSame( $tag, webp_uploads_img_tag_update_mime_type( $tag, 'the_content', $attachment_id ) );
 	}
+
+	/**
+	 * Tests whether additional mime types generated only for allowed image sizes or not.
+	 *
+	 * @test
+	 */
+	public function it_should_create_mime_types_for_allowed_sizes_only() {
```

Comment:
See below: If we add another test case using the image sizes global, we may want to rename this to
```suggestion
	public function it_should_create_mime_types_for_allowed_sizes_only_via_filter() {
```

Reply:
Updated.